data/brands/amenity/fast_food.json

```
@@ -795,6 +796,7 @@
         "brand:wikidata": "Q465751",
         "brand:wikipedia": "en:Chipotle Mexican Grill",
         "cuisine": "mexican",
+        "diet:vegetarian": "yes",
```

```suggestion
        "diet:vegan": "yes",
```
Chipotle's sofritas is vegan as well as no-protein options. This would encompass vegetarian.

---

I wouldn't replace vegetarian with vegan due to the way certain searches may work but I'd certainly add it.